packages/terra-demographics-banner/tests/nightwatch/demographics-banner-spec.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-unused-expressions */
+
+const screenshot = require('terra-toolkit').screenshot;
+
+module.exports = {
+  afterEach: (browser, done) => {
```

Comment:
It looks like these updates were checked into the generator. When you checked out the new generator did you first uninstall the old one?
https://github.com/cerner/generator-terra-module/pull/30
```
npm uninstall -g generator-terra-module
```
If this isn't run before installing a new version it won't pull the updates